src/engraving/rendering/dev/systemlayout.cpp

```diff
@@ -854,12 +854,18 @@ void SystemLayout::layoutSystemElements(System* system, LayoutContext& ctx)
                         // add tremolo to skyline
                         if (e->isChord() && toChord(e)->tremolo()) {
                             Tremolo* t = toChord(e)->tremolo();
-                            Chord* c1 = t->chord1();
-                            Chord* c2 = t->chord2();
-                            if (!t->twoNotes() || (c1 && !c1->staffMove() && c2 && !c2->staffMove())) {
+                            if (!t->twoNotes()) {
                                 if (t->chord() == e && t->addToSkyline()) {
```

Comment:
I think `t->chord() == e &&` is also meaningless for single-chord tremolos; for two-chord tremolos, it makes sense because it ensures that the tremolo is only added to the skyline when we see it for the first time (i.e. via its first chord). But single-chord tremolos are seen only once anyway. 
(I'd recommend to remove this logic soon, because if someone sees it in the future and isn't completely aware of the history of this code, they might be confused by it. But doing it in a next PR is also fine)